src/test/java/gov/healthit/chpl/aqa/stepDefinitions/FuzzyMatchTextSteps.java

```diff
@@ -18,14 +20,21 @@
 
     private WebDriver driver;
     private static final int DELAY = 20;
+    private String url = System.getProperty("url");
 
+    /**
+     * Constructor creates new driver.
+     */
     public FuzzyMatchTextSteps() {
         driver = Hooks.getDriver();
+        if (StringUtils.isEmpty(url)) {
+            url = "http://localhost:3000/";
+        }
     }
 
     @Given("^I am on listing details page \"([^\"]*)\"$")
     public void iAmOnListingDetailsPage(final String arg1) throws Throwable {
```

Comment:
If you write the Step in the Feature file as `@Given("^I am on the Details page of Listing \"([^\"]*)\"$")` then this Step should use the Step from the SEDTweaksStep.java file, and we won't have virtually identical steps

Reply:
It is cucumber limitation by design, step definitions are glued to feature file and matched based on the regexp(that is why i wrote it that way) and having same step in features is resulting in 'duplicate' error. I am going to look for workaround and probably go for a separate stepdefinition file to define common steps, just not sure of it yet.